tests/v2/validation/observability/rbac/stackstate_rbac_test.go

```diff
@@ -0,0 +1,257 @@
+	require.NoError(rb.T(), err)
+
+	systemProject, err := projects.GetProjectByName(client, rb.cluster.ID, systemProject)
+	require.NoError(rb.T(), err)
+	systemProjectID := strings.Split(systemProject.ID, ":")[1]
+
+	err = charts.InstallStackstateAgentChart(standardClient, rb.stackstateAgentInstallOptions, rb.stackstateConfigs.ClusterApiKey, rb.stackstateConfigs.Url, systemProjectID)
+	require.NoError(rb.T(), err)
+	log.Info("Stackstate agent chart installed successfully")
+
+	rb.T().Log("Verifying the deployments of stackstate agent chart to have expected number of available replicas")
+	err = extencharts.WatchAndWaitDeployments(client, rb.cluster.ID, charts.StackstateNamespace, meta.ListOptions{})
+	require.NoError(rb.T(), err)
+
+	rb.T().Log("Verifying the daemonsets of stackstate agent chart to have expected number of available replicas nodes")
+	err = extencharts.WatchAndWaitDaemonSets(client, rb.cluster.ID, charts.StackstateNamespace, meta.ListOptions{})
+	require.NoError(rb.T(), err)
+}
+
+func (rb *StackStateRBACTestSuite) TestMembersCannotInstallStackstate() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
+
+	client, err := rb.client.WithSession(subSession)
+	require.NoError(rb.T(), err)
+
+	initialStackstateAgent, err := extencharts.GetChartStatus(client, rb.cluster.ID, charts.StackstateNamespace, charts.StackstateK8sAgent)
+	require.NoError(rb.T(), err)
+
+	if initialStackstateAgent.IsAlreadyInstalled {
+		rb.T().Skip("Stack state agent is already installed, skipping the tests.")
+	}
+
+	tests := []struct {
+		name   string
+		role   rbac.Role
+		member string
+	}{
+		{"Cluster Member", rbac.ClusterMember, rbac.StandardUser.String()},
+		{"Project Owner", rbac.ProjectOwner, rbac.StandardUser.String()},
+		{"Project Member", rbac.ProjectMember, rbac.StandardUser.String()},
+	}
+
+	for _, tt := range tests {
+		var newUser *management.User
+		user, err := users.CreateUserWithRole(rb.client, users.UserConfig(), tt.member)
+		require.NoError(rb.T(), err)
+		newUser = user
+		rb.T().Logf("Created user: %v", newUser.Username)
+
+		standardClient, err := rb.client.AsUser(user)
+		require.NoError(rb.T(), err)
+
+		if strings.Contains(tt.role.String(), "project") {
+			err := users.AddProjectMember(rb.client, rb.project, user, tt.role.String(), nil)
+			require.NoError(rb.T(), err)
+		} else {
+			err := users.AddClusterRoleToUser(rb.client, rb.cluster, user, tt.role.String(), nil)
+			require.NoError(rb.T(), err)
+		}
+
+		systemProject, err := projects.GetProjectByName(client, rb.cluster.ID, systemProject)
+		require.NoError(rb.T(), err)
+		systemProjectID := strings.Split(systemProject.ID, ":")[1]
+
+		err = charts.InstallStackstateAgentChart(standardClient, rb.stackstateAgentInstallOptions, rb.stackstateConfigs.ClusterApiKey, rb.stackstateConfigs.Url, systemProjectID)
+		require.Error(rb.T(), err)
+		k8sErrors.IsForbidden(rb.T(), err.Error())
```

Comment:
why do you need both of these?

Reply:
it might sometimes happen that cluster member will be able to install chart for whatever reason when an error is expected but we end up getting nil. second one is to make sure its indeed a forbidden error.